crates/config/src/lib.rs

```diff
@@ -391,6 +394,9 @@ pub struct Config {
     /// If disabled, it is possible to access artifacts which were not recompiled or cached.
     pub unchecked_cheatcode_artifacts: bool,
 
+    /// Soldeer dependencies
+    pub dependencies: Option<BTreeMap<String, SoldeerDependency>>,
```

Comment:
I want this to be a new type that wraps the betreemap

Reply:
Updated, does that work? or maybe i misunderstood?